Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Use the existing ScienceBeam parser image as a base
+FROM elifesciences/sciencebeam-parser:latest
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the current directory contents into the container /app folder
+COPY . /app
+
+# Install dependencies in requirements.txt
+RUN pip install --upgrade pip \
+    && pip install -r requirements.txt
+
+# Expose port 80 outside this container
+EXPOSE 80
```

Comment:
if the parser is running in the container (it's at 8070) then we won't need to expose it.